spiketools/sim/trials.py

```diff
@@ -80,3 +81,327 @@ def sim_trials_poisson(n_trials, rate_pre, rate_post, time_pre=1, time_post=2, r
 SPIKETRIAL_FUNCS = {
     'poisson' : sim_trials_poisson,
 }
+
+
+def sim_trial_placefield(height_mean, height_std, width_mean, width_std, place_loc_mean, place_loc_std, n_bins, noise_std, base_mean, base_std,
```

Comment:
make sure line length and spacing for function definitions is properly formatted - for example:
```
def sim_trial_placefield(height_mean, height_std, width_mean, width_std, place_loc_mean,
                         place_loc_std, n_bins, noise_std, base_mean, base_std,
                         n_trials, vary_height=True, vary_width=True, vary_place_loc=True,
                         presence_ratio=0.6):
```

Check this here and throughout